### PR TITLE
Operator Docs changes related to Unity features

### PR DESCRIPTION
**Pull request by panigs7**
# Description
Added operator related documents for volume health monitor feature for unity. Also updated support matrix.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/installation/operator/unity.md [hunk 1]
```diff
@@ -138,3 +138,38 @@ kubectl edit configmap -n unity unity-config-params
...
   2. "Kubelet config dir path" is not yet configurable in case of Operator based driver installation.
   3. Also, snapshotter and resizer sidecars are not optional to choose, it comes default with Driver installation. 
+
+## Volume Health Monitoring
+This feature is introduced in CSI Driver for unity version 2.1.0.
+
+### Operator based installation
+
+Volume Health Monitoring feature is optional and by default this feature is disabled for drivers when installed via operator.
+To enable this feature, we will have to add the below block to the driver manifest before installing the driver. This ensures to install external health monitor sidecar. To get the volume health state `value` under controller should be set to true as seen below. To get the volume stats `value` under node should be set to true.
```
Comment (karthikk92) [inline content/docs/csidriver/installation/operator/unity.md:148]: can we also add command "kubectl edit csiunity/unity -n namespace_of_driver" and modify bellow parameters?

**Review by karthikk92 — APPROVED** [review]

File: content/docs/csidriver/release/unity.md [hunk 2]
```diff
@@ -25,3 +23,8 @@ description: Release notes for Unity CSI driver
...
 | NFS Clone - Resize of the snapshot is not supported by Unity Platform.| Currently, when the driver takes a clone of NFS volume, it succeeds. But when the user tries to resize the NFS volumesnapshot, the driver will throw an error. The user should never try to resize the cloned NFS volume.|
 | Delete namespace that has PVCs and pods created with the driver. The External health monitor sidecar crashes as a result of this operation.| Deleting the namespace deletes the PVCs first and then removes the pods in the namespace. This brings a condition where pods exist without their PVCs and causes the external-health-monitor sidecar to crash. This is a known issue and has been reported at https://github.com/kubernetes-csi/external-health-monitor/issues/100|
+
+### Note:
```
Comment (shanmydell) [inline content/docs/csidriver/release/unity.md:27]: This has to go across all drivers

**Review by shanmydell — CHANGES_REQUESTED** [review]

File: content/docs/csidriver/installation/operator/_index.md [hunk 3]
```diff
@@ -19,21 +19,21 @@ If you have installed an old version of the `dell-csi-operator` which was availa
 #### Full list of CSI Drivers and versions supported by the Dell CSI Operator
 | CSI Driver         | Version   | ConfigVersion  | Kubernetes Version   | OpenShift Version     |
 | ------------------ | --------- | -------------- | -------------------- | --------------------- |
-| CSI PowerMax       | 1.6       | v5             | 1.18, 1.19, 1.20     | 4.6, 4.7              |
 | CSI PowerMax       | 1.7       | v6             | 1.19, 1.20, 1.21     | 4.6, 4.7              |
 | CSI PowerMax       | 2.0.0     | v2.0.0         | 1.20, 1.21, 1.22     | 4.6 EUS, 4.7, 4.8     |
-| CSI PowerFlex      | 1.4       | v4             | 1.18, 1.19, 1.20     | 4.6, 4.7              |
+| CSI PowerMax       | 2.1.0     | v2.1.0         | 1.20, 1.21, 1.22     | 4.8, 4.9              |
```
Comment (prasar6) [inline content/docs/csidriver/installation/operator/_index.md:24]: @panigs7 - Openshift version should be **4.8, 4.8(EUS) and 4.9**

**Review by prasar6 — CHANGES_REQUESTED** [review]

File: content/docs/csidriver/installation/operator/_index.md (hunk 3 above)
Comment (panigs7) [reply content/docs/csidriver/installation/operator/_index.md]: Done

File: content/docs/csidriver/release/unity.md (hunk 2 above)
Comment (panigs7) [reply content/docs/csidriver/release/unity.md:27]: Added the note across all drivers supporting Volume health monitor and RWOP access mode features.

**Review by rensyct — DISMISSED** [review]

**Review by rensyct — DISMISSED** [review]

**Review by prasar6 — APPROVED** [review]

File: content/docs/csidriver/installation/operator/unity.md (hunk 1 above)
Comment (prasar6) [inline content/docs/csidriver/installation/operator/unity.md:148]: To enable this feature, ~~we will have to~~ add the

File: content/docs/csidriver/release/powerflex.md [hunk 4]
```diff
@@ -24,3 +24,7 @@ There are no fixed issues in this release.
 | Issue | Workaround |
 |-------|------------|
 | Delete namespace that has PVCs and pods created with the driver. The External health monitor sidecar crashes as a result of this operation.| Deleting the namespace deletes the PVCs first and then removes the pods in the namespace. This brings a condition where pods exist without their PVCs and causes the external-health-monitor sidecar to crash. This is a known issue and has been reported at https://github.com/kubernetes-csi/external-health-monitor/issues/100|
+
+### Note:
+
+- This release we have added support for kurbernetes alpha features like Volume Health Monitoring and RWOP (ReadWriteOncePod) access mode, however, these features will not be available in Openshift environment as Openshift doesn't allow enabling of alpha features through feature-gates for Production Grade clusters.
```
Comment (prasar6) [inline content/docs/csidriver/release/powerflex.md:30]: Support for kurbernetes alpha features like Volume Health Monitoring and RWOP (ReadWriteOncePod) access mode introduced in the release will not be available in Openshift environment as Openshift doesn't support enabling of alpha features for Production Grade clusters.

File: content/docs/csidriver/release/powerscale.md [hunk 5]
```diff
@@ -25,3 +25,7 @@ There are no fixed issues in this release.
...
 | If some older NFS exports /terminated worker nodes still in NFS export client list, CSI driver tries to add a new worker node it fails (For RWX volume). | User need to manually clean the export client list from old entries to make successful additon of new worker nodes.
 | Delete namespace that has PVCs and pods created with the driver. The External health monitor sidecar crashes as a result of this operation.| Deleting the namespace deletes the PVCs first and then removes the pods in the namespace. This brings a condition where pods exist without their PVCs and causes the external-health-monitor sidecar to crash. This is a known issue and has been reported at https://github.com/kubernetes-csi/external-health-monitor/issues/100|
+
+### Note:
+
+- This release we have added support for kurbernetes alpha features like Volume Health Monitoring and RWOP (ReadWriteOncePod) access mode, however, these features will not be available in Openshift environment as Openshift doesn't allow enabling of alpha features through feature-gates for Production Grade clusters.
```
Comment (prasar6) [inline content/docs/csidriver/release/powerscale.md:31]: Support for kurbernetes alpha features like Volume Health Monitoring and RWOP (ReadWriteOncePod) access mode introduced in the release will not be available in Openshift environment as Openshift doesn't support enabling of alpha features for Production Grade clusters.

File: content/docs/csidriver/release/powerstore.md [hunk 6]
```diff
@@ -22,3 +22,7 @@ There are no fixed issues in this release.
 | Issue                                                        | Resolution or workaround, if known                           |
 | ------------------------------------------------------------ | ------------------------------------------------------------ |
 | Delete namespace that has PVCs and pods created with the driver. The External health monitor sidecar crashes as a result of this operation | Deleting the namespace deletes the PVCs first and then removes the pods in the namespace. This brings a condition where pods exist without their PVCs and causes the external-health-monitor sidecar to crash. This is a known issue and has been reported at https://github.com/kubernetes-csi/external-health-monitor/issues/100 <br>|
+
+### Note:
+
+- This release we have added support for kurbernetes alpha features like Volume Health Monitoring and RWOP (ReadWriteOncePod) access mode, however, these features will not be available in Openshift environment as Openshift doesn't allow enabling of alpha features through feature-gates for Production Grade clusters.
```
Comment (prasar6) [inline content/docs/csidriver/release/powerstore.md:28]: Support for kurbernetes alpha features like Volume Health Monitoring and RWOP (ReadWriteOncePod) access mode introduced in the release will not be available in Openshift environment as Openshift doesn't support enabling of alpha features for Production Grade clusters.

File: content/docs/csidriver/release/unity.md [hunk 7]
```diff
@@ -25,3 +23,7 @@ description: Release notes for Unity CSI driver
...
+
+- This release we have added support for kurbernetes alpha features like Volume Health Monitoring and RWOP (ReadWriteOncePod) access mode, however, these features will not be available in Openshift environment as Openshift doesn't allow enabling of alpha features through feature-gates for Production Grade clusters.
```
Comment (prasar6) [inline content/docs/csidriver/release/unity.md:29]: Support for kurbernetes alpha features like Volume Health Monitoring and RWOP (ReadWriteOncePod) access mode introduced in the release will not be available in Openshift environment as Openshift doesn't support enabling of alpha features for Production Grade clusters.

**Review by karthikk92 — DISMISSED** [review]

**Review by prasar6 — CHANGES_REQUESTED** [review]
Rephrase the statements indicated

File: content/docs/csidriver/installation/operator/unity.md (hunk 1 above)
Comment (panigs7) [reply content/docs/csidriver/installation/operator/unity.md]: Done

File: content/docs/csidriver/release/powerflex.md (hunk 4 above)
Comment (panigs7) [reply content/docs/csidriver/release/powerflex.md]: Done

File: content/docs/csidriver/release/powerscale.md (hunk 5 above)
Comment (panigs7) [reply content/docs/csidriver/release/powerscale.md]: Done

File: content/docs/csidriver/release/powerstore.md (hunk 6 above)
Comment (panigs7) [reply content/docs/csidriver/release/powerstore.md]: Done

File: content/docs/csidriver/release/unity.md (hunk 7 above)
Comment (panigs7) [reply content/docs/csidriver/release/unity.md]: Done

**Review by rensyct — DISMISSED** [review]

File: content/docs/csidriver/installation/operator/unity.md [hunk 8]
```diff
@@ -138,3 +138,38 @@ kubectl edit configmap -n unity unity-config-params
...
+
+    controller:
+      envs:
+      # X_CSI_ENABLE_VOL_HEALTH_MONITOR: Enable/Disable health monitor of CSI volumes from Controller plugin - volume condition.
+      # Install the 'external-health-monitor' sidecar accordingly.
+      # Allowed values:
+      #   true: enable checking of health condition of CSI volumes
+      #   false: disable checking of health condition of CSI volumes
+      # Default value: false
+      - name: X_CSI_ENABLE_VOL_HEALTH_MONITOR
```
Comment (bandak2) [inline content/docs/csidriver/installation/operator/unity.md:163]: should this be indented 2 more spaces Spandita ?

**Review by prasar6 — DISMISSED** [review]

File: content/docs/csidriver/installation/operator/unity.md (hunk 8 above)
Comment (panigs7) [reply content/docs/csidriver/installation/operator/unity.md]: Fixed it. Thanks @bandak2 !!

**Review by rensyct — APPROVED** [review]

**Review by bandak2 — APPROVED** [review]

**Review by karthikk92 — APPROVED** [review]

**Review by shanmydell — APPROVED** [review]
LGTM

**Pull request merged**